docs/running from source.md

```diff
@@ -8,7 +8,7 @@ use that version. These steps are for developers or platforms without compiled r
 
 What you'll need:
  * [Python 3.8.7 or newer](https://www.python.org/downloads/), not the Windows Store version
-   * **Python 3.12 is currently unsupported**
+   * Python 3.12 is currently the newest supported version
```

Comment:
There's another one on line 33 of this same file.

Reply:
Completely missed that. Thanks